Viewer/src/net/relinc/viewer/GUI/HomeController.java

```diff
@@ -886,6 +962,38 @@ private void saveSessionButtonFired()
 		SPOperations.writeStringToFile(session.getJSONString(this), sessionFile.getPath());
 		sampleDirectoryGUI.fillSessionsListView();
 	}
+
+	@FXML
+	private void saveTrimmedDataClicked() {
+		if(Dialogs.showConfirmationDialog("Confirm", "Please confirm", "Are you sure you want to save this trim to disk?", stage)) {
+			getCheckedSamples().forEach(sample -> {
+				sample.DataFiles.forEach(df -> {
+					df.dataSubsets.forEach(ds -> {
+						if(ds.getEndTemp() != null) {
+							ds.setEnd(ds.getEndTemp());
```

Comment:
so this means you can't "reset" trim after saving it to disk, which isn't all that unexpected imo

Reply:
Note how many index scalings are happening here... `getEndTemp()` scales the original index to a user index, and then `setEnd()` takes it in as a user index and scales it back to an original index. Just a note to help understand how it works